.changeset/metal-pans-breathe.md

```diff
@@ -0,0 +1,26 @@
+---
+'braid-design-system': minor
+---
+
+---
+updated:
+  - TooltipRenderer
+---
+
+**TooltipRenderer:** Add `placement` prop, support `top` and `bottom`. Set default placement to `top`.
+
+**EXAMPLE USAGE**
+
+```tsx
+<TooltipRenderer
+  id={id}
+  tooltip={<Text>This is a tooltip!</Text>}
+  placement="top"
```

Comment:
Suggest we make the example different from the default value.
```suggestion
  placement="bottom"
```

Reply:
Nice catch. This wasn't the default when I wrote it.